_includes/openring_template_out.html

```diff
@@ -0,0 +1,91 @@
+
```

Comment:
Since this file is dynamically generated, there is no reason to commit it. Just add it to the `.gitigore` file.

Reply:
Did you address my previous comment? Should this file be part of the .gitignore?